### PR TITLE
feat: add archway

### PR DESCRIPTION
**Pull request by ericHgorski**

### CODE REVIEW

**Comment by terencelimzhengwei** [comment]
@ericHgorski - added counterparty chains and updated the axelar channels because it seemed wrong

File: chains/mainnet/cosmoshub.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ module.exports = {
     'kaiyo-1': 'channel-343',
     'osmosis-1': 'channel-141',
     'stride-1': 'channel-391',
+    'cosmoshub-1': 'channel-623',
```
Comment (ericHgorski) [inline chains/mainnet/cosmoshub.js:25]: Is there a need for IBC channels to the chain the assets originate on?

**Comment by ericHgorski** [comment]
@terencelimzhengwei The build fails because we are missing bi-directional channels for Axelar, cosmos hub, Kujira and Osmosis. These aren't currently listed in the chain registry.

**Comment by terencelimzhengwei** [comment]
lol there was some merge issues - but resolved now

**Pull request merged**